test/integration/kube_api_discovery_test.go

```diff
@@ -399,3 +402,109 @@ func TestGetAPIResourceList(t *testing.T) {
 		})
 	}
 }
+
+// safe to run in parallel with serial tests since it only reads CRDs, see main_test.go.
+func TestCRDAdditionalPrinterColumns_Parallel(t *testing.T) {
```

Comment:
I would expect to see an allowlist per CRD that lists all the fields (so that if a new field is added to existing CRD, we remember to decide if it needs a printer column or not).

Reply:
Because I was under the impression that columns should only include the most important/identifying fields (see other comment below), that implies that we would most likely not change the columns when we add new fields, so such a test felt like extra work and maybe annoying to keep up-to-date.

However, this test will fail when we add a new CRD, to force us to pick the columns for the new CRD.

Reply:
I was hoping to make that decision at the addition of each field (who knows what we will add in the future), but I can get behind this reasoning.